File: Go/1550_Three_Consecutive_Odds/solution.go
package solution

func threeConsecutiveOdds(arr []int) bool {
	for i := 0; i < len(arr)-2; i++ {
		if arr[i]*arr[i+1]*arr[i+2]%2 == 1 {
			return true
		}
	}

	return false
}

File: JavaScript/1550_Three_Consecutive_Odds/solution.js
/**
 * @param {number[]} arr
 * @return {boolean}
 */
const threeConsecutiveOdds = function (arr) {
    let counter = 0;
    for (let i = 0; i < arr.length; i++) {
        switch (arr[i] % 2) {
            case 1:
                counter++;
                break;
            case 0:
                counter = 0;
        };

        if (counter > 2) return true;
    }

    return false;
};